.eslintrc.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   },
   plugins: ['flowtype', 'prettier'],
   rules: {
-    'prettier/prettier': 'error',
+    'prettier/prettier': 'off',  // disabled temporarily to prevent un-necessary delta in conversion to TS.
```

Comment:
> My vote is to keep this unchanged ('error'), as long as we keep our change progressive, I'd rather live with the whitespace diff (which we could probably turn off during diffing) than inconsistent coding style.

May I suggest that we keep it 'off' in this MR and follow-up with a "quick" whitespace-only MR that'll be an almost automatic approval?

<br/>

> I didn't see any typescript specific support (@typescript-eslint/xxx). Should we add those? Not sure if it can live in the same config with "flow", worst case separate config?

👍  It's a small repo. Should we go with the recommendations? I don't know how much these diverge from our coding style. If it's bad, maybe it should follow the whitespace MR. ;). https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/TYPED_LINTING.md

Reply:
Cool, happy to leave Prettier on.

Regarding @typescript-eslint/xxx linting rules/presets, I have been playing with the linting recently.  Hopefully I'll get that up shortly.